pkg/security/apiserver/admission/sccadmission/admission.go

```diff
@@ -242,8 +242,8 @@ func shouldIgnore(a admission.Attributes) (bool, error) {
 }
 
 // SetSecurityInformers implements WantsSecurityInformer interface for constraint.
-func (c *constraint) SetSecurityInformers(informers securityinformer.SharedInformerFactory) {
-	c.sccLister = informers.Security().InternalVersion().SecurityContextConstraints().Lister()
+func (c *constraint) SetSecurityInformers(informers securityv1informer.SharedInformerFactory) {
+	c.sccLister = informers.Security().V1().SecurityContextConstraints().Lister()
```

Comment:
@deads2k setting a v1 lister here means that we'll have to use the external security API in the FindApplicableSCCs method in this file, since we'll now be getting `*v1.SecurityContextConstraints`. The `FindApplicableSCCs` method is used in https://github.com/openshift/origin/blob/master/pkg/security/apiserver/registry/podsecuritypolicyreview/rest.go#L76 but I don't think that file is meant to consume the external api

Reply:
Duplicate.